README.md

```diff
@@ -1 +1,156 @@
-# JS-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1>MooTV Web Application</h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+## 📗 Table of Contents
+
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 live-demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 📖 MooTV <a name="about-project"></a>
+
+MooTV is currently a single page website, which uses the TV Maze open api to get data about the movies and series. Everyone can check the shows and movies displayed on the website and give likes and comments accordingly.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML & CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Fetching data from an API to display a list of TV Shows**
+- **DOM manipulation with API data**
+- **Give likes and comments to every movie**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live demo <a name="live-demo"></a>
+
+- [Video Presentation](https://drive.google.com/file/d/1-YtxC95Cn8p672IHB-ZiY4i7h8Eai5dS/view?usp=sharing)
+
+- [Live Demo](https://anbehindy.github.io/JS-Capstone/dist)
```

Comment:
[OPTIONAL] It looks like your live demo is not deployed as expected. Kindly add correct demo link or just add message about link is not available right now.